cpp/src/groupby/hash/groupby.cu

```diff
@@ -55,10 +58,16 @@ namespace {
  * @brief List of aggregation operations that can be computed with a hash-based
  * implementation.
  */
-constexpr std::array<aggregation::Kind, 7> hash_aggregations{
+constexpr std::array<aggregation::Kind, 10> hash_aggregations{
     aggregation::SUM, aggregation::MIN, aggregation::MAX,
     aggregation::COUNT_VALID, aggregation::COUNT_ALL,
-    aggregation::ARGMIN, aggregation::ARGMAX};
+    aggregation::ARGMIN, aggregation::ARGMAX,
+    aggregation::MEAN, aggregation::STD, aggregation::VARIANCE};
```

Comment:
I still do not agree that `MEAN`, `STD`, and `VARIANCE` should be listed here. An aggregation should be broken into it's simple components before checking against `is_hash_aggregation`.

Reply:
I am not clear on this.
The name `is_hash_aggregation` denotes if an aggregation is supported by hash based groupby. It does not indicate whether it's simple or compound aggregation.

Reply:
@jrhemstad need a response on this please.